x/airdrop/types/events.go

```diff
@@ -1,5 +1,8 @@
 package types
 
 const (
-	EventTypeClaim = "airdrop claim"
+	EventTypeClaim            = "airdrop claim"
```

Comment:
do events need to be underscore rather than space delimited? I'm not sure where these are used?

Reply:
yes indeed it should be underscore...